src/content/en/updates/posts/2016/03/access-usb-devices-on-the-web.markdown

```diff
@@ -0,0 +1,370 @@
+[feedback on the implementation](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3EUSB).
+
+The WebUSB API is currently planned to be enabled experimentally on your
+origin in Origin Trials, and available now locally on your machine using an
+experimental flag. The implementation is partially complete and currently
+available on Chrome OS, Linux, Mac, and Windows. Go to
+`chrome://flags/#enable-webusb`, enable the highlighted flag, restart Chrome
+and you should be good to go.
+
+<img style="width:723px; max-height:205px" src="/web/updates/images/2016-03-02-access-usb-devices-on-the-web/web-usb-flag.png" alt="Web USB Flag highlighted in chrome://flags"/>
+
+## Privacy & Security
+
+### Attacks against USB devices
+
+The WebUSB API does not even try to provide a way for a web page to connect to
+arbitrary USB devices. There are plenty of published attacks against USB
+devices that makes it unsafe to allow this.
+
+For this reason, a USB device can define a set of origins that are allowed to
+connect to it. This is similar to the
+[CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS)
+mechanism in HTTP. In other words, **WebUSB devices are associated with a web
+origin** and can only be accessed from a page from the same origin.
+
+Hardware manufacturers will have to update the firmware in their USB devices in order
+to enable WebUSB access to their device via the [Platform Capability
+descriptor](https://wicg.github.io/webusb/#webusb-platform-capability-descriptor).
+Later a [Public Device Registry](https://wicg.github.io/webusb/#public-device-registry) 
+will be created so that hardware manufacturers can support WebUSB on existing
+devices.
+
+### Available for Origin Trials
+
+In order to get as much feedback as possible from developers using the WebUSB
+API in the field, we will also add this feature in Chrome 54 as an
+[origin trial](https://github.com/jpchase/OriginTrials) for Chrome
+OS, Linux, Mac, and Windows. During the origin trial, the API may still change in
+backward-incompatible ways before we freeze it into the web platform.  To use
+this experimental API in Chrome with no flag, you'll need to [request a token
+for your origin](http://bit.ly/WebUSBOriginTrial) and [insert it in your
+application](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-guide.md).
+
+The trial will end in March 2017. By that point, we expect to have figured
+out any changes necessary to stabilize the feature and move it out from Origin
+Trials.
+
+### HTTPS Only
+
+Because this experimental API is a powerful new feature added to the Web,
+Chrome aims to make it available only to [secure
+contexts](https://w3c.github.io/webappsec/specs/powerfulfeatures/#intro). This
+means you'll need to build with TLS in mind.
+
+> We care deeply about security, so you will notice that new Web capabilities
+require HTTPS. The WebUSB API is no different, and is yet another good reason
```

Comment:
I've been asked to point to this page instead of http://www.chromium.org/Home/chromium-security/deprecating-powerful-features-on-insecure-origins previously.